std_msgs/Header header
PersonVelocity[] people
